modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path

```diff
@@ -365,6 +365,123 @@
 	<td></td>
 </tr>
 
+<tr>
+	<td>RULE_BUILDER_FORM_VIEW_NAME</td>
+	<td>//input[contains(@placeholder,'Untitled Form View')]</td>
+	<td></td>
+</tr>
+
+<!--TABS-->
+<tr>
+	<td>RULES_TAB</td>
+	<td>//button[@id='rules']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FIELDS_TAB</td>
+	<td>//button[@id='fields']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_NAME_TEXT_FIELD</td>
+	<td>//input[@placeholder='Untitled Rule']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_ADD_BUTTON</td>
+	<td>//div[contains(@class,'multi-panel-sidebar')]//button[@class='btn btn-monospaced btn-primary']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>RULE_BUILDER_NAME</td>
+	<td>//input[@placeholder='Untitled Rule']</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>CREATE_NEW_RULE_BUTTON</td>
+	<td>//button[contains(text(),'Create New Rule')]</td>
+	<td></td>
+</tr>
+
+<tr>
```

Comment:
Hi @calvinctvo , please update the following selectors to be compatible with the previous script you created AddRulesViaFormView, there you were editing a rule without passing the rule name since you had a single rule, but now you need to identify the rule by its name, so the selector should be improved to something like this, please try it out to see if it works for both scripts:

`//button[contains(@class,'collapse') and contains(@aria-expanded,'false')]//span[contains(.,'${ruleName}')] | //button[contains(@class,'collapse') and contains(@aria-expanded,'false')]`

So, if the macro uses a name, it will match in the first part of the selector, if not, it will match the second part.

Thanks!